tensorflow_addons/layers/tests/octave_convolutional_test.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for octave convolutional layers."""
+
+import numpy as np
+import pytest
+
+from tensorflow import keras
+from tensorflow_addons.layers.octave_convolutional import OctaveConv1D
+
+
+@pytest.mark.usefixtures("maybe_run_functions_eagerly")
```

Comment:
I don't think this is needed. There is not crazy stuff happenning in the `call` that would require us checking autograph's output.